packages/website/pages/login.js

```diff
@@ -1,15 +1,22 @@
 import { useState } from 'react'
 import Router from 'next/router'
+import { useQueryClient } from 'react-query'
 import { loginEmail, loginSocial } from '../lib/magic.js'
 import Button from '../components/button.js'
-import { useQueryClient } from 'react-query'
+import GithubIcon from '../icons/github.js'
 
+/**
+ * Static Props
+ *
+ * @returns {{ props: import('../components/types.js').LayoutProps}}
+ */
 export function getStaticProps() {
   return {
     props: {
       title: 'Login - Web3 Storage',
       redirectTo: '/files',
       redirectIfFound: true,
+      pageBgColor: 'bg-w3storage-red-dark',
```

Comment:
This should be using the color `w3storage-red` according to Figma

Reply:
You're right good catch